CommonWallet/Core/Classes/Modules/TransactionDetails/TransactionDetailsPresenter.swift

```diff
@@ -50,15 +50,21 @@ extension TransactionDetailsPresenter: TransactionDetailsPresenterProtocol {
     }
 
     func performAction() {
-        let receiverInfo = ReceiveInfo(accountId: transactionData.peerId,
-                                       assetId: transactionData.assetId,
-                                       amount: nil,
-                                       details: nil)
+        let receiverInfo = transactionData.status != .rejected ?
```

Comment:
I think that we just need to override the action itself. This implement is too specific to application